Makefile

```diff
@@ -44,6 +44,11 @@ godeps = $(shell $(call godeps_cmd,$(1)))
 build: generate-always ## Build main binary
 	CGO_ENABLED=0 time go build -ldflags "-X $(version_pkg).gitCommit=$(git_commit) -X $(version_pkg).builtAt=$(built_at)" ./cmd/eksctl
 
+# Build binaries for Linux, Windows and Mac and place them in dist/
+.PHONY: build-all
+build-all: generate-always
```

Comment:
build-release? Not a strong opinion..

Reply:
It doesn't do a release build though, in that the `release` tag isn't set. I used `all` to emphasise that it builds for all major OSes.